FirelandsProject/firelands-cata
# Feature: added missing spell scripts

### Changes proposed

- added missing script registration

### Issues addressed


- Closes

### SOURCE:


### Tests performed


- In game test

### How to Test the Changes:


1. Download the branch
2. execute sql update
3. run the worldserver
4. check the db errors section

### Known issues and TODO list


---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
